Sources/TuistKit/Generator/ManifestTargetGenerator.swift

```diff
@@ -34,13 +34,15 @@ class ManifestTargetGenerator: ManifestTargetGenerating {
                       filesGroup: .group(name: "Manifest"))
     }
 
-    func manifestTargetBuildSettings() throws -> [String: String] {
+    // MARK: - Private
+
+    private func manifestTargetBuildSettings() throws -> [String: SettingValue] {
         let frameworkParentDirectory = try resourceLocator.projectDescription().parentDirectory
-        var buildSettings = [String: String]()
-        buildSettings["FRAMEWORK_SEARCH_PATHS"] = frameworkParentDirectory.pathString
-        buildSettings["LIBRARY_SEARCH_PATHS"] = frameworkParentDirectory.pathString
-        buildSettings["SWIFT_INCLUDE_PATHS"] = frameworkParentDirectory.pathString
-        buildSettings["SWIFT_VERSION"] = Constants.swiftVersion
+        var buildSettings = [String: SettingValue]()
+        buildSettings["FRAMEWORK_SEARCH_PATHS"] = .string(frameworkParentDirectory.pathString)
```

Comment:
Is the `.string` necessary considering that the model conforms the `ExpressiveByLiteral...` protocols?

Reply:
Yeah, unfortunately it's necessary as `frameworkParentDirectory.pathString` is a string value, not a string literal. AFAIK it won't work without the explicit enum case :(